example/workflows/errorsAndRetries.yml

```diff
@@ -33,7 +33,8 @@ HelloWorldFailWorkflow:
       Type: Task
       Resource: ${HelloWorldLambdaFunction.Arn}
       Retry:
-        - ErrorEquals:
+        - &LambdaServiceExceptionRetry
```

Comment:
This line was deleted because it defines a template that is no longer referenced anywhere else in this file, so it can be deleted.